FAQs.md

```diff
@@ -0,0 +1,121 @@
+## FREQUENTLY ASKED QUESTIONS
+
+This is the list of Frequently asked questions for Jockey music player. It is intended to serve as a self-service utility document. This is to aid new users or already existing users of Jockey music player.
+This document may be updated from time to time to suit any new changes to the software as new developments or changes will be implemented.
+
+1. **What is Jockey Music Player?**
+
+Jockey is a music player for Android based on Google's Material Design standards. The point of Jockey is to be a simple, lightweight media player with features designed for music enthusiasts and casual listeners alike. 
+
+
+2. **What are the features of jockey music player?**
+
+Jockey music player is designed with the essential features of a music player and they include;
+
+- playlist
+- theme customization
+- equalizer
+- sleep timer
+- search tool
+
+3. **How can I navigate between songs, artists, album, genres and playlists?**
+
+You can navigate to these pages simply by swiping left or right depending on the position.
+
+4. **How do I create a create a playlist?**
+
+To create a playlist on Jockey, navigate to the playlist tab by swiping left or right. On the playlist, tab click the plus (+) sign below and two options will pop up Smart Playlist and Playlist. Click the plus sign beside playlist, input playlist name and click on create.
+
+5. **What is smart playlist?**
+
+Smart playlist as the name implies is a playlist that automatically sorts and add music to the list according to customized rules or parameters set. 
+
+6. **Can I import and export playlists on Jockey music player?**
+
+No, playlists can neither be imported nor exported.
```

Comment:
It may be a good idea to clarify that standard (i.e. non-smart) playlists are accessible to other apps.